src/tests/testHttpRequest.cc

```diff
@@ -44,13 +44,12 @@ void
 testHttpRequest::testCreateFromUrl()
 {
     /* vanilla url, implicit method */
-    unsigned short expected_port;
     SBuf url("http://foo:90/bar");
     const auto mx = MasterXaction::MakePortless<XactionInitiator::initHtcp>();
     HttpRequest *aRequest = HttpRequest::FromUrl(url, mx);
-    expected_port = 90;
+    AnyP::KnownPort expected_port = 90;
```

Comment:
We cannot (easily) use AnyP::Port here because CPPUNIT_ASSERT_EQUAL()s below would not be able to "print" that. However, since the hard-coded port is indeed known in this case, this `KnownPort` usage is not wrong.

Reply:
Please check that situation after adding an `AnyP::Port` stream print operator.

FWIW, I am okay with this as-is.

Reply:
I am skipping this secondary change request until its [parent](https://github.com/squid-cache/squid/pull/1255/files#r1095576128) change request is resolved.